src/components/hits/components/NoCtaCard.jsx

```diff
@@ -6,16 +6,16 @@ import { framerMotionHits } from '@/config/animationConfig';
 
 // Display an image, within a framer-motion wrapper
 const NoCtaCard = ({ hit }) => {
+  const style = {height: "100%"}
```

Comment:
Shouldn't this be a CSS class for ease of customisation and understanding in future?